src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -80,4 +81,50 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+  
+  /** Returns the total number of messages for all users. */
+  public int getTotalMessageCount(){
+    Query query = new Query("Message");
+<<<<<<< HEAD
```

Comment:
These indicators show merge conflicts which should be resolved otherwise the code won't compile or fail at runtime.